cli/ballerina-cli/src/main/resources/cli-help/ballerina-grpc.help

```diff
@@ -21,11 +22,13 @@ OPTIONS
            definition. If a package is not specified, the output will be written
            to a 'temp' directory in the current location.
 
-       --mode client | service | proxy
+       --mode client | service
            Set the `client` or `service` mode to generate sample code.
-           Set the `proxy` mode to generate a gateway proxy.
            If not specified, only the stub file is generated.
 
+       --proto_path <path>
```

Comment:
```suggestion
       --proto-path <path>
```

Reply:
Option names use dashes

Reply:
we referred to the protoc command and use the same option key. https://developers.google.com/protocol-buffers/docs/proto3#generating. We will change it to dash

Reply:
POSIX command generally uses dashes, hence the suggestion. :)

Reply:
Added a task https://github.com/ballerina-platform/ballerina-standard-library/issues/2590